packages/network-controller/src/create-network-client.ts

```diff
@@ -153,7 +153,6 @@ function createNetworkAndChainIdMiddleware({
 }) {
   return createScaffoldMiddleware({
     eth_chainId: ChainId[network],
-    net_version: NetworkId[network],
```

Comment:
I see the changelog entry for this:
> CHANGED: For Built-in Infura Networks, net_version is no longer locally resolved by the scaffold middleware

but isn't this a breaking change?

Reply:
Why did even ever have middleware for this method? Shouldn't it come for free on the infura node?

Reply:
I'm not sure if this counts as breaking or not. The json-rpc interface doesn't change, thus no changes are required from downstream consumers unless calling this is breaking on the infura side. After we remove `networkVersion`, there really shouldn't be that many `net_version` calls originating from the wallet itself.

Reply:
OK so the call would still get resolved, just not by the middleware?

Reply:
Right. Previously `net_version` to infura built-in networks would be resolved offline in middleware. Now it will result in a rpc request to the provider